rules/execute-command.js

```diff
@@ -64,9 +75,7 @@ function runCommands(command, args) {
     if (command === 'run') {
       command = args.shift();
     }
-    runCommand(`pre${command}`);
     runCommand(command, args);
-    runCommand(`post${command}`);
```

Comment:
Are these jazelle postcommand hooks? We rely on postcommand hooks in web-code to track perf

Reply:
no these are things like `prebuild/postbuild` in package.json scripts, which we ended up inlining for all projects during the migration anyway

Reply:
What about postinstall? That is likely the most widely used post-hook.

Reply:
Yarn v2 supports postinstall: https://yarnpkg.com/advanced/lifecycle-scripts

Reply:
The install command is handled a bit differently and doesn't go through the web_binary rule. Yarn2 supports postinstall directly, so it works for free.